Fix game issues

MoxElliot opened the pull request:
- Correct answer disables after first click, enables after confetti
- Fixed oval api import, added circle, square, triangle, removed rectagle
- Changed end screen to read "Your Score is: x/y points!"
- Added "Loading..." and jumping dots between flashcards

MoxElliot committed:
pulled to update

MoxElliot committed:
added/removed/fixed shapes

Corrected oval import, added square, triangle, circle, removed rectangle

MoxElliot committed:
correct answer button disabled

correct answer button now disables onClick via setState, resets to enabled after setTimeout runs via setState again

MoxElliot committed:
refactored score screen

total score now displays as 'Your Score is: x/y points!'

MoxElliot committed:
created GameSpinnerWindow

MoxElliot committed:
added comments in api.jsx to figure this out

MoxElliot committed:
Merge branch 'fix_game-issues' of https://github.com/chingu-voyages/v42-geckos-team-22 into feature_enable-loading-spinner

MoxElliot committed:
Adjusted async in api.jsx to add loading state

MoxElliot committed:
loading window works

Loading screen shows between flashcards, needs centering

MoxElliot committed:
loading screen works

E-wang42 commented:
Does not show game landing page after starting a game and switching to another from the homepage links. 
This does not happen when switching games from the drop down menu. 

- Please add "handleToggleGame" function to homepage "Link" elements similar to the dropDown menu component.
- Loading spinner sometimes does not show.  Duplicate on loading spinner function with another merged PR.

![gameLanding](https://user-images.githubusercontent.com/91093545/218289502-276b305b-b591-4ad4-baef-808ea87df9a7.gif)

MoxElliot committed:
merge conflict resolved

MoxElliot committed:
Home Page to game Fix

Added the handleToggleGames, useDisptach, useSelector suite from DropDown.jsx to HomePage links. Now the game displays loading/landing page as expected when user clicks home page links

MoxElliot committed:
Error fix

removed empty dependency array in GameLoadingWindow useEffect. Fixed bad set state call error in console

E-wang42 commented:
Clicking the home Link buttons now brings up the mobile menu instead.  

![bug](https://user-images.githubusercontent.com/91093545/218332223-0959d409-7279-4ffe-b216-2be46372e708.gif)

MoxElliot committed:
Removed toggleMenu from home screen links

E-wang42 reviewed (approved):
good change
